bot/__init__.py

```diff
@@ -55,7 +55,7 @@ def monkeypatch_trace(self, msg, *args, **kwargs):
 
 
 logging.basicConfig(
-    format="%(asctime)s pd.beardfist.com Bot: | %(name)30s | %(levelname)8s | %(message)s",
+    format="%(asctime)s pd.beardfist.com Bot: | %(name)33.33s | %(levelname)8s | %(message)s",
```

Comment:
I've not seen this kind of formatting before. What does `33.33` do?

Reply:
It's probably something that I should reverse, now that I think of it. It's width + truncation. However, full information is probably more useful than alignment.

Reply:
I've changed it to 33s. That accommodates for the watch channel log names (`bot.cogs.watchchannels.talentpool`), but won't truncate if something somehow generates an even longer logger name.